teach/static/admin/js/toggle_clubs_guides_translation.js

```diff
@@ -0,0 +1,27 @@
+(function() {
+    var $ = django.jQuery.bind(django.jQuery);
+
+    function allowChoosingTranslation() {
+        $(".field-category").hide();
+        $(".field-translation_of").show();
+    }
+
+    function disallowChoosingTranslation() {
+        $(".field-category").show();
+        $(".field-translation_of").hide();
+    }
+
+    document.addEventListener("DOMContentLoaded", function() {
+        var isTranslationYes = $("#id_is_translation_0");
+        var isTranslationNo = $("#id_is_translation_1");
+
+        if(isTranslationYes.is(":checked")) {
```

Comment:
The `initial` option is meant to specify the initial value only. So, if you are editing a guide instead of creating one, the `initial` value is overridden in the `__init__` function.